This file contains two minor updates to dist3 PL50 that were used
to generage Configure.
    --Andy Dougherty		doughera@lafcol.lafayette.edu


Index: Oldconfig.U
Prereq:  3.0.1.6 
*** /home2/doughera/lib/dist/U/Oldconfig.U	Tue Jan 31 10:31:26 1995
--- Oldconfig.U	Thu Feb  2 14:33:32 1995
***************
*** 180,192 ****
  
  		case "$1" in
  		aix) osname=aix
  			tmp=`( (oslevel) 2>/dev/null || echo "not found") 2>&1`
  			case "$tmp" in
! 			'not found') osvers=3.2.0 ;;
  			'<3240'|'<>3240') osvers=3.2.0 ;;
  			'=3240'|'>3240'|'<3250'|'<>3250') osvers=3.2.4 ;;
  			'=3250'|'>3250') osvers=3.2.5 ;;
! 			*) osvers='' ;;
  			esac
  			;;
  		dnix) osname=dnix
--- 180,198 ----
  
  		case "$1" in
  		aix) osname=aix
+ ?X: aix 4.1 uname -a output looks like
+ ?X:		AIX  foo  1 4 000123456789   
+ ?X: where $4 is the major release number and $3 is the (minor) version.
+ ?X: More detail on the version is available with the oslevel command.
+ ?X: in 3.2.x, it output a string (see case statements below).  In 4.1,
+ ?X: it puts out something like  4.1.1.0
  			tmp=`( (oslevel) 2>/dev/null || echo "not found") 2>&1`
  			case "$tmp" in
! 			'not found') osvers="$4"."$3" ;;
  			'<3240'|'<>3240') osvers=3.2.0 ;;
  			'=3240'|'>3240'|'<3250'|'<>3250') osvers=3.2.4 ;;
  			'=3250'|'>3250') osvers=3.2.5 ;;
! 			*) osvers=$tmp;;
  			esac
  			;;
  		dnix) osname=dnix
Index: mansrc.U
Prereq:  3.0.1.6 
*** /home2/doughera/lib/dist/U/mansrc.U	Tue Jan 31 10:31:56 1995
--- mansrc.U	Thu Feb  2 14:34:36 1995
***************
*** 1,5 ****
  ?RCS: $Id: mansrc.U,v 3.0.1.6 1995/01/30 14:39:34 ram Exp $
- ?RCS:
  ?RCS: Copyright (c) 1991-1993, Raphael Manfredi
  ?RCS: 
  ?RCS: You may redistribute only under the terms of the Artistic Licence,
--- 1,4 ----
***************
*** 74,80 ****
  '')
  	lookpath="$prefixexp/man/man1 $prefixexp/man/u_man/man1"
  	lookpath="$lookpath $prefixexp/man/l_man/man1"
! 	lookpath="$lookpath /usr/local/man/man1 /usr/local/man/man1 /usr/man/manl"
  	lookpath="$lookpath /usr/man/local/man1 /usr/man/l_man/man1"
  	lookpath="$lookpath /usr/local/man/u_man/man1 /usr/local/man/l_man/man1"
  	lookpath="$lookpath /usr/man/man.L"
--- 73,79 ----
  '')
  	lookpath="$prefixexp/man/man1 $prefixexp/man/u_man/man1"
  	lookpath="$lookpath $prefixexp/man/l_man/man1"
! 	lookpath="$lookpath /usr/local/man/man1 /opt/man/man1 /usr/man/manl"
  	lookpath="$lookpath /usr/man/local/man1 /usr/man/l_man/man1"
  	lookpath="$lookpath /usr/local/man/u_man/man1 /usr/local/man/l_man/man1"
  	lookpath="$lookpath /usr/man/man.L"
